apps/testingaccessibility/src/pages/welcome/index.tsx

```diff
@@ -177,7 +133,8 @@ const Header: React.FC = () => {
         alt="a shining lighthouse"
       />
       <h1 className="font-bold lg:text-5xl sm:text-4xl text-3xl font-heading">
-        Welcome to Testing Accessibility
+        {upgrade ? `You've Upgraded ` : `Welcome to `}
+        {process.env.NEXT_PUBLIC_SITE_TITLE}
```

Comment:
small change here to acknowledge what happened

Reply:
![Screen Shot 2022-07-06 at 15 27 25](https://user-images.githubusercontent.com/86834/177653802-6acea6c7-5f80-4622-8d44-72f4bbbb4b3f.png)